cgi-bin/LJ/Session.pm

```diff
@@ -950,8 +950,8 @@ sub get_cookie_path {
 
 sub get_path_user {
     my ( $path ) = @_;
-    return unless $path =~ m!^/(\w{1,25})\b!;
-    # FIXME: username length should be a named constant (#1568)
+    my $exp = '^/(\w{1,' . $LJ::USERNAME_MAXLENGTH . '})\b';
+    return unless $path =~ /$exp/;
```

Comment:
You don't need to break out the variable from the regular expression. It can be interpreted inline by Perl just fine:
`m!^/(\w{1,$LJ::USERNAME_MAXLENGTH})\b!`